services/forest/src/service/forestLP.js

```diff
@@ -66,12 +67,7 @@ module.exports = (ForestLPPersistence, restAPI) => {
           forestLPColorSet(category),
         );
       } catch (e) {
-        const error = {
-          code: 500,
-          stack: e.stack,
-          message: 'Error retrieving layer',
-        };
-        throw error;
+        throw new RestifyErrors.InternalError('Error retrieving layer');
```

Comment:
this is ok, but you didn't change the error on line 40

Reply:
Done